omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/FormResource1_8.java

```diff
@@ -194,7 +197,11 @@ protected NeedsPaging<Form> doGetAll(RequestContext context) throws ResponseExce
 	 */
 	@Override
 	protected NeedsPaging<Form> doSearch(RequestContext context) {
-		return new NeedsPaging<Form>(Context.getFormService().getForms(context.getParameter("q"), false), context);
+		String fuzzyName = context.getParameter("q");
```

Comment:
You do not even need these variables. You can just directly use the context calls in the getForms API

Reply:
True, but to be readable and easy to understand at a glance I thought of extracting the to variables.